src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -100,12 +101,16 @@ class ContextualSaveBar extends React.PureComponent<CombinedProps, State> {
       />
     );
 
+    const logoMarkup = alignContentFlush ? null : (
```

Comment:
`alignContentFlush` doesn't default to true so null will be shown by default making this a breaking change unless i'm missing something?

Reply:
I'm not sure now if I'm misreading it. But I think...

- If the flag isn't specified, it will be `falsey`, showing the logo markup as it has before.

- If the flag is specified, it will be `truthy`, resulting in no logo section thus the content will grow.

- If the flag is specified and specifically set to `false`, the logo markup will show.